hyperspy/tests/component/test_gaussianhf.py

```diff
@@ -19,9 +19,12 @@
 
 import numpy as np
 from numpy.testing import assert_allclose
+import pytest
 
 from hyperspy.components1d import GaussianHF
 from hyperspy.signals import Signal1D
+from hyperspy.tests.component.test_components import _generate_parameters
```

Comment:
I think that it'll be better for readability to redefine the suggested ``TRUE_FALSE_2_TUPLE `` variable Instead of importing the ``_generate_parameters`` function from a different module.